tests/parser/parserTests.txt

```diff
@@ -0,0 +1,17 @@
+!! Version 2
+
+# Use the test runner to ensure that the extension is loaded
+!! functionhooks
+remote_version
+!! endfunctionhooks
+
+!! test
+mediawiki.org version - starts with 1.
+Used to ensure that the parser hook is registered
+!! wikitext
+{{padleft:|2|{{#remote_version:https://www.mediawiki.org/w/api.php}}}}
```

Comment:
I feel that this is more of an integration test than a parser test and it doesn't seem right. In my opinion, it doesn't make much sense to use parser tests to test the extension parser functions. Therefore, I think it would be better if we didn't have parser tests for these functions at all